maven-plugin/src/main/java/org/openmrs/maven/plugins/CreateProject.java

```diff
@@ -281,14 +300,13 @@ private void createModule() throws MojoExecutionException {
 			    "2.4");
 			archetypeArtifactId = SDKConstants.REFAPP_ARCH_ARTIFACT_ID;
 		} else if (TYPE_CONTENT_PACKAGE.equals(type)) {
-			contentpackage = wizard.promptForValueIfMissingWithDefault("What version of the content package (-D%s) do you want to support?", contentpackage, "contentpackage", "1.0.0");
 			archetypeArtifactId = SDKConstants.CONTENT_PACKAGE_ARCH_ARTIFACT_ID;
 		} else {
 			throw new MojoExecutionException("Invalid project type");
 		}
 
 		archetypeVersion = getSdkVersion();
-		packageName = "org.openmrs.module." + artifactId;
+		packageName = TYPE_CONTENT_PACKAGE.equals(type) ? "org.openmrs.content." + artifactId : "org.openmrs.module." + artifactId;
```

Comment:
Maybe we refactor this into a function call or using the above:

```java
packageName = groupId + "." + artifactId
```

That said, we shouldn't _need_ to prompt for a package for a content package since it shouldn't have code.